libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -342,7 +358,7 @@ AP_GPS_UBLOX::read(void)
                 _delay_time = 750;
             }
         } else {
-            _delay_time = 2000;
+            _delay_time = 4000;
```

Comment:
Was this overloading the bandwidth to the GPS? (Just curious) I could see it being a larger concern with having to provide correction data now.